regions_www/tabs/manual_body.html

```diff
@@ -14,10 +15,14 @@
     <div class="col-md-9">
 
       <div class="col-md-12 text-left" style="padding-top:10px;">
-        <h3>User Manual for the Propensity to Cycle Tool (PCT)</h3>
-        <p>The material here was developed during 2016 and 2017 for use in training. To request more information or provide feedback, please email us at <a href="mailto:pct@pct.bike?Subject=Manual">pct@pct.bike</a> or report issues on our <a target="_blank" href="https://github.com/npct/pct-shiny/issues">GitHub issue tracker</a>. The PCT also has a <a target="_blank" href="https://blog.pct.bike/">blog</a>, which contains further information on some of the topics featured in the case studies and FAQs.  Contributions to GitHub, the blog or other parts of the project are welcome.</p>
+        <h3>How to use the Propensity to Cycle Tool (PCT)</h3>
+        <p>The material here was developed during 2016 and 2019 for use in training. To request more information or provide feedback, please email us at <a href="mailto:pct@pct.bike?Subject=Manual">pct@pct.bike</a> or report issues on our <a target="_blank" href="https://github.com/npct/pct-shiny/issues">GitHub issue tracker</a>. The PCT also has a <a target="_blank" href="https://blog.pct.bike/">blog</a>, which contains further information on some of the topics featured in the case studies and FAQs.  Contributions to GitHub, the blog or other parts of the project are welcome.</p>
 
         <div style="padding-top:6px;">
+          <h4><a name="essentials"></a>PCT Essentials</h4>
+          <ul style="line-height:2.2;">
+          <li>Read our two pager <a target="_blank" href="https://cdn.rawgit.com/npct/pct-shiny/30b98621b37da3cb66fcf2253583af9f026f9fce/regions_www/www/static/03d_other_reports/2020-pct-essentials.pdf">Essentials</a> document first, for a high level summary</li>
```

Comment:
same here.

Also is this meant to be:
>Read our two pager

Reply:
Thanks @nikolai-b 
I wanted to use it but for a test I had used raw git. Now have updated the links.